src/tools/rbd_mirror/ImageSyncThrottler.cc

```diff
@@ -88,33 +90,35 @@ void ImageSyncThrottler<I>::start_sync(
   }
 
   if (start) {
-      sync_holder_ctx->m_sync->send();
+    sync_holder_ctx->m_sync->send();
   }
 }
 
 template <typename I>
-void ImageSyncThrottler<I>::cancel_sync(const std::string& local_image_id) {
+void ImageSyncThrottler<I>::cancel_sync(librados::IoCtx &local_io_ctx,
```

Comment:
Just to abstract away how the key is formed -- I originally wanted to change the signature to pass in ImageCtxT but that would have involved more work in bootstrap to only cancel if an image sync was scheduled.